official/README.md

```diff
@@ -2,34 +2,64 @@
 
 The TensorFlow official models are a collection of example models that use TensorFlow's high-level APIs. They are intended to be well-maintained, tested, and kept up to date with the latest TensorFlow API. They should also be reasonably optimized for fast performance while still being easy to read.
 
+These models are used as integration tests, ensuring that the models run with the same speed and performance with each new TensorFlow build.
```

Comment:
nit based on minutiae of testing semantics: technically, I would say those are end-to-end tests, since it involves hardware, data transfer, etc. etc.

Reply:
Good to know the more accurate term. Fixed